### ISSUE TITLE
Feature request: add support for "boxed" columns

### ISSUE THREAD

**Issue by noib3** [kamiyaa/joshuto#39]
Would it be possible to add support for boxed columns, i.e. surronding each column in its own box drawn with `┌`, `─`, `┐`, etc.. characters? Here's how it looks in lf:

![2021-Jan-30@01:34:43](https://user-images.githubusercontent.com/59321248/106340834-604c5400-629b-11eb-90ae-5901e12491df.png)

**Comment by kamiyaa** [comment]
added with 8d6a3b4d58c2380613cfc0896520b8e0523533ea

You can enable it in `joshuto.toml` via
```toml
show_borders = true
```
One thing to note is it doesn't draw as nice of a rectangle. It looks more like:

```
+---------+
|  |  |   |
+---------+
```
rather than
```
+--+--+---+
|  |  |   |
+--+--+---+
```
Feel free to reopen if needed

**Comment by noib3** [comment]
It doesn't compile on my machine:

```shell
$ cargo install --path=. --force
  Installing joshuto v0.8.5 (/Users/noibe/joshuto)
    Updating crates.io index
   Compiling joshuto v0.8.5 (/Users/noibe/joshuto)
warning: unused imports: `Paragraph`, `Wrap`
 --> src/ui/widgets/tui_worker.rs:4:20
  |
4 | use tui::widgets::{Paragraph, Widget, Wrap};
  |                    ^^^^^^^^^          ^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

error[E0004]: non-exhaustive patterns: `&CopyFileName` not covered
  --> src/commands/key_command.rs:73:15
   |
17 | / pub enum KeyCommand {
18 | |     BulkRename,
19 | |     ChangeDirectory(path::PathBuf),
20 | |     CommandLine(String, String),
...  |
25 | |     CopyFileName,
   | |     ------------ not covered
...  |
68 | |     TabSwitch(i32),
69 | | }
   | |_- `KeyCommand` defined here
...
73 |           match self {
   |                 ^^^^ pattern `&CopyFileName` not covered
   |
   = help: ensure that all possible cases are being handled, possibly by adding wildcards or more match arms
   = note: the matched value is of type `&KeyCommand`

warning: unused variable: `clipboard`
  --> src/commands/file_ops.rs:79:14
   |
79 |         for (clipboard, command) in clipboards.iter() {
   |              ^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_clipboard`
   |
   = note: `#[warn(unused_variables)]` on by default

warning: unused variable: `show_icons`
  --> src/fs/entry.rs:24:42
   |
24 |     pub fn from(direntry: &fs::DirEntry, show_icons: bool) -> std::io::Result<Self> {
   |                                          ^^^^^^^^^^ help: if this is intentional, prefix it with an underscore: `_show_icons`

warning: unused variable: `x`
  --> src/util/input.rs:96:46
   |
96 |         MouseEvent::Press(MouseButton::Left, x, y)
   |                                              ^ help: if this is intentional, prefix it with an underscore: `_x`

warning: variable does not need to be mutable
  --> src/fs/metadata.rs:52:13
   |
52 |         let mut mimetype = None;
   |             ----^^^^^^^^
   |             |
   |             help: remove this `mut`
   |
   = note: `#[warn(unused_mut)]` on by default

error: aborting due to previous error; 5 warnings emitted

For more information about this error, try `rustc --explain E0004`.
error: failed to compile `joshuto v0.8.5 (/Users/noibe/joshuto)`, intermediate artifacts can be found at `/Users/noibe/joshuto/target`

Caused by:
  could not compile `joshuto`

To learn more, run the command again with --verbose.
```

**Comment by kamiyaa** [comment]
Should be fixed with 7be7c13a739e0b43f10af91d4a062e60da13260c
